[TST] Upcoming dependency test failures

The weekly build with nightly wheels from numpy and pandas
has failed. Check the logs for any updates that need to be
made in matplotlib.
https://github.com/matplotlib/matplotlib/actions/runs/3163174215

This does look real:

```
>       assert cb.ax.get_ylim() == (1e-4, 1e6)
190
E       assert (9.9999999999...05, 1000000.0) == (0.0001, 1000000.0)
191
E         At index 0 diff: 9.999999999999999e-05 != 0.0001
192
E         Full diff:
193
E         - (0.0001, 1000000.0)
194
E         + (9.999999999999999e-05, 1000000.0)
```

--- a/lib/matplotlib/tests/test_contour.py
+++ b/lib/matplotlib/tests/test_contour.py
@@ -4,7 +4,8 @@ import re
 
 import contourpy
 import numpy as np
-from numpy.testing import assert_array_almost_equal
+from numpy.testing import (
+    assert_array_almost_equal, assert_array_almost_equal_nulp)
 import matplotlib as mpl
 from matplotlib.testing.decorators import image_comparison
 from matplotlib import pyplot as plt, rc_context
@@ -313,7 +314,7 @@ def test_contourf_log_extension():
     cb = plt.colorbar(c1, ax=ax1)
     assert cb.ax.get_ylim() == (1e-8, 1e10)
     cb = plt.colorbar(c2, ax=ax2)
-    assert cb.ax.get_ylim() == (1e-4, 1e6)
+    assert_array_almost_equal_nulp(cb.ax.get_ylim(), np.array((1e-4, 1e6)))
     cb = plt.colorbar(c3, ax=ax3)
 
 

--- a/lib/matplotlib/tests/test_streamplot.py
+++ b/lib/matplotlib/tests/test_streamplot.py
@@ -34,7 +34,8 @@ def test_startpoints():
     plt.plot(start_x, start_y, 'ok')
 
 
-@image_comparison(['streamplot_colormap'], remove_text=True, style='mpl20')
+@image_comparison(['streamplot_colormap'], remove_text=True, style='mpl20',
+                  tol=0.022)
 def test_colormap():
     X, Y, U, V = velocity_field()
     plt.streamplot(X, Y, U, V, color=U, density=0.6, linewidth=2,

